translations/ja/LC_MESSAGES/messages.po

```diff
@@ -161,7 +161,7 @@ msgstr ""
 
 #: /content/includes/menu.yaml
 msgid "About"
-msgstr ""
+msgstr "AMP とは"
```

Comment:
This seems to translate to "What is AMP".  Is there a better translation for just "About"?